app.js

```diff
@@ -0,0 +1,19 @@
+const express = require(`express`);
+const { getCategories } = require("./controllers/games.controllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
we only need this for POST requests.  remove it for now and add it back when it is needed.